issuer.yml

```
@@ -53,7 +55,8 @@ paths:
         - oAuth2: []
         - zCap: []
       operationId: updateCredentialStatus
-      x-expectedCaller: "Issuer Service"
+      x-expectedCaller:
+        - Issuer Service
```

Hmm, this seems like it would be the issuer coordinator that would use this endpoint (and that the endpoint would reside on the Status Service).

```suggestion
        - Issuer Coordinator
```